join_request/olanod.yml

```
@@ -0,0 +1,29 @@
+collective: Potoc
+member:
+  name: Daniel Olano
+  address: 15oLanodWWweiZJSoDTEBtrX7oGfq6e8ct5y5E6fVRDPhUgj
+  github: olanod
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 15oLanodWWweiZJSoDTEBtrX7oGfq6e8ct5y5E6fVRDPhUgj
  matrix:
  github: olanod
```